packages/rspack/src/logging/truncateArgs.ts

```diff
@@ -87,4 +87,4 @@ const truncateArgs = (args, maxLength) => {
 	});
 };
 
-export { truncateArgs };
+export = truncateArgs;
```

Comment:
this is a little bit tricky, why not use` import {  truncateArgs } from './logging/truncateArgs'`

Reply:
Fixed